dlib/simd/simd4i.h

```diff
@@ -148,7 +191,9 @@ namespace dlib
     inline simd4i operator* (const simd4i& lhs, const simd4i& rhs) 
     { 
 #ifdef DLIB_HAVE_SSE41
-        return _mm_mullo_epi32(lhs, rhs); 
+        return _mm_mullo_epi32(lhs, rhs);
```

Comment:
The old version has a trailing blank. The new version has this trailing blank dropped (presumably by Xcode).